### DISCUSSION TITLE
FrPatient Patient.name.UsualName

### DISCUSSION THREAD

**Discussion by Lammchi** [Interop-Sante/hl7.fhir.fr.core#41] [Q&A]
Bonjour tout le monde, 

J'aurais une question sur un cas d'usage assez précis. 

Par exemple : La personne a un nom de naissance, prénom(s) de naissance, et un prénom d'usage mais il n'a pas de nom de d'usage. Est-ce que mon retour FHIR doit se composer de cette manière ou bien dans ce cas là, si la personne a un prénom d'usage mais pas de nom d'usage, nous devons retourner le nom d'usage = nom de naissance ou est-ce que le retour ci-dessous est correct ?

        "name": [
          {
            "use": "official",
            "family": "LENOMDENAISSANCE",
            "given": [
              "PREMIER",
              "AUTRES PRENOMS"
            ]
          },
          {
            "use": "usual",
            "family": "",
            "given": [
              "UTILISE"
            ]
          }
        ],

Cordialement,

**Comment by ypoiron** [comment]
Bonjour, le `family` n'est pas obligatoire pour un nom utilisé (slice usualName)
